### DISCUSSION TITLE
How to modify a web page?

### DISCUSSION THREAD

**Discussion by Bulua** [langgenius/dify#9260] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I started the dify project using docker compose. I want to modify the front-end page, for example, hide the ExploreNav option. How should I do it?
![image](https://github.com/user-attachments/assets/28faddb1-a566-416b-a8b7-8f5268b83203)

**Comment by taowang1993** [comment]
Why would you want to do that?

That Explore Nav is at:

```
web/app/components/header/explore-nav/index.tsx
```
Just comment out the code or delete it. 

Then docker build the dify-web image.

**Reply by fisherever** [reply]
Then docker build the dify-web image.
===
Could you kindly tell me how to achieve this step?
